### PR TITLE
OP note re: notifications on delegate answer

### PR DESCRIPTION
**Pull request by NicholasBurkhammer**
Added note (formatting may be required) that notifications for when a delegate answers a call on behalf of a delegator.

Note:  Notifications when a delegate answers a call on behalf of a delegator is not available in Cloud Voicemail.  Users can receive notifications for missed calls.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/set-up-phone-system-voicemail.md [hunk 1]
```diff
@@ -56,6 +56,8 @@ The following information is about configuring Cloud Voicemail to work with on-p
 
 6. To enable Voicemail features such as customizing greetings, and visual voicemail in Skype for Business clients, connectivity from Office 365 to the Exchange server mailbox via Exchange Web Services is required. To enable this connectivity you must configure the new Exchange Oauth authentication protocol described in [Configure OAuth authentication between Exchange and Exchange Online organizations](https://technet.microsoft.com/library/dn594521(v=exchg.150).aspx), or run the Exchange Hybrid Wizard from Exchange 2013 CU5 or greater. Additionally, you must configure integration and Oauth between Skype for Business Online and Exchange server described in [Configure Integration and OAuth between Skype for Business Online and Exchange Server](https://docs.microsoft.com/skypeforbusiness/deploy/integrate-with-exchange-server/oauth-with-online-and-on-premises). 
 
+Note:  Notifications when a delegate answers a call on behalf of a delegator is not available in Cloud Voicemail.  Users can receive notifications for missed calls.
```
Comment (JohanFreelancer9) [inline Teams/set-up-phone-system-voicemail.md]:
```suggestion
> [!NOTE]
> Notifications when a delegate answers a call on behalf of a delegator is not available in Cloud Voicemail. Users can receive notifications for missed calls.
```

Comment (NicholasBurkhammer) [reply Teams/set-up-phone-system-voicemail.md]: Looks fine to me.

**Comment by scanum** [comment]
@dstrome  PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**